trimesh/visual/color.py

```diff
@@ -939,8 +939,8 @@ def uv_to_interpolated_color(uv, image):
     img = np.asanyarray(image.convert('RGBA'))
 
     colors00 = img[y_floor, x_floor]
-    colors01 = img[y_ceil, x_floor]
-    colors10 = img[y_floor, x_ceil]
+    colors01 = img[y_floor, x_ceil]
+    colors10 = img[y_ceil, x_floor]
```

Comment:
a00 and a01 both use (1-dy) for Y interpolation - they also have to use the same y coordinates for the texture